### PR TITLE
Update MTU Local Zone Day 2

### PR DESCRIPTION
**Pull request by yunjiang29**
Updates on **aws-ipi-localzone-byo-subnet-day2** job
* Rename `aws-ipi-localzone-byo-subnet-day2*` to `aws-ipi-localzone-byo-subnet-ovn-day2*`
* Set MTU to `1200` (OVN)

### CODE REVIEW

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-debug

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-localzone-byo-subnet-day2-f14

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-localzone-byo-subnet-day2-f14

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-localzone-byo-subnet-day2-f14

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/day2/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-day2-provision-chain.yaml [hunk 1]
```diff
@@ -8,12 +8,14 @@ chain:
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-aws-custom-vpc
   - ref: ipi-conf-aws
+  - ref: ovn-conf
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/day2/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-day2-provision-chain.yaml:11]: we have to explicitly set the OVN networking type?

Comment (yunjiang29) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/day2/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-day2-provision-chain.yaml:11]: @anuragthehatter suggested adding this step, it's required by `network-mtu-migration-decrease`

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-localzone-byo-subnet-day2-f14

**Comment by yunjiang29** [comment]
test  https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/43179/rehearse-43179-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-localzone-byo-subnet-day2-f14/1713760718672629760/artifacts/aws-ipi-localzone-byo-subnet-day2-f14/network-mtu-migration-decrease-end/build-log.txt
```
[2023-10-16T04:39:38+00:00] Getting the current cluster network MTU...
++ oc get network.config '--output=jsonpath={.items..status.clusterNetworkMTU}'
+ cluster_mtu=1200
+ '[' -z 1200 ']'
+ log 'Cluster network MTU is 1200'
++ date -Is
+ echo '[2023-10-16T04:39:39+00:00] Cluster network MTU is 1200'
[2023-10-16T04:39:39+00:00] Cluster network MTU is 1200
```

**Comment by yunjiang29** [comment]
@jianlinliu PTAL

**Comment by jianlinliu** [comment]
> @jianlinliu PTAL

My only comment is https://github.com/openshift/release/pull/43179/files#r1322666057

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/day2/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-day2-provision-chain.yaml (hunk 1 above)
Comment (yunjiang29) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/day2/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-day2-provision-chain.yaml:11]: Updated.

**Comment by yunjiang29** [comment]
> My only comment is https://github.com/openshift/release/pull/43179/files#r1322666057

Sorry I missed that, updated now. PTAL again, thanks

**Comment by jianlinliu** [comment]
cc @liangxia to review

**Pull request merged**